src/webprojects/relay.md

```diff
@@ -0,0 +1,80 @@
+# Firefox Relay: how to localize and test
+
+Firefox Relay⁩ email masks protect your true email address from public view, automatically forwarding messages to your true inbox. Now you can receive only the messages you want in your inbox. For Canada and the US, phone mask feature as a premium service is offered.
+
+Firefox Relay consists of two parts:
+* [Firefox Relay Website](https://pontoon.mozilla.org/projects/firefox-relay-website)
+* [Firefox Relay Add-on](https://pontoon.mozilla.org/projects/firefox-relay-add-on)
+
+## Firefox Relay Website
+
+This project is open to localization in all languages. The free version with a limit of 5 email masks is available to all users. However, Relay Premium, a paid service, is available in [these markets](https://docs.google.com/document/d/1ww1ab5Vhr4AISMlWZvTe0cnz8GD16F8xE5vegcJZNwQ/edit#) at the moment. The team will continue market expansion as well as adding the product features and making improvements.
+
+Phone masking, a paid service, is offered to limited markets. Localizable content is limited in languages where the markets are offered the Relay Premium services.
+
+* Production: https://relay.firefox.com/
```

Comment:
These links show up as text in the rendered version, you shouldn't rely on automatic conversion

```suggestion
* Production: [https://relay.firefox.com](https://relay.firefox.com/)
```